src/main/java/mate/academy/Main.java

```diff
@@ -44,12 +56,30 @@ public static void main(String[] args) {
         yesterdayMovieSession.setMovie(fastAndFurious);
         yesterdayMovieSession.setShowTime(LocalDateTime.now().minusDays(1L));
 
-        MovieSessionService movieSessionService = null;
         movieSessionService.add(tomorrowMovieSession);
         movieSessionService.add(yesterdayMovieSession);
 
         System.out.println(movieSessionService.get(yesterdayMovieSession.getId()));
         System.out.println(movieSessionService.findAvailableSessions(
                         fastAndFurious.getId(), LocalDate.now()));
+
+        UserService userService = (UserService) injector.getInstance(UserService.class);
+        AuthenticationService authenticationService = new AuthenticationServiceImpl(userService);
+
+        User newRegisteredUser = null;
+        try {
+            System.out.println("=========================================");
+            newRegisteredUser = authenticationService.register("mary@mail.com", "111111");
+        } catch (Exception e) {
+            throw new RegistrationException("",e);
```

Comment:
The `catch` block is catching a generic `Exception`, which is not a good practice. It's better to catch specific exceptions that can be thrown by the `register` method. This will make the code more robust and easier to debug.

Reply:
ok, done